### PR TITLE
[EOC-216] BUG can't remove list or cohort description

### PR DESCRIPTION
**Pull request by alemikolo**

### CODE REVIEW

File: client/src/modules/cohort/components/CohortHeader.jsx [hunk 1]
```diff
@@ -48,43 +48,49 @@ class CohortHeader extends PureComponent {
...
     }
 
-    if (
-      isNameInputVisible &&
-      nameInputValue.trim().length >= 1 &&
-      isClickedOutside
-    ) {
-      this.handleNameUpdate();
+    if (isNameInputVisible && isClickedOutside) {
+      if (_trim(nameInputValue).length >= 1) {
```
Comment (marekrozmus) [inline client/src/modules/cohort/components/CohortHeader.jsx]: could be just `> 0`

File: client/src/modules/cohort/components/CohortHeader.jsx [hunk 2]
```diff
@@ -48,43 +48,49 @@ class CohortHeader extends PureComponent {
...
     const { code } = event;
 
-    if (code === KeyCodes.ENTER || code === KeyCodes.ESCAPE) {
+    if (code === KeyCodes.ESCAPE) {
       const action = isDescriptionTextareaVisible
         ? this.handleDescriptionUpdate
         : this.handleNameUpdate;
 
       action();
     }
```
Comment (marekrozmus) [inline client/src/modules/cohort/components/CohortHeader.jsx]: should return as if the `code === ESCAPE` then we don't need to check other values

File: client/src/modules/list/components/ListHeader.jsx [hunk 3]
```diff
@@ -76,13 +76,17 @@ class ListHeader extends PureComponent {
...
     const { code } = event;
 
-    if (code === KeyCodes.ENTER || code === KeyCodes.ESCAPE) {
+    if (code === KeyCodes.ESCAPE) {
       const action = isDescriptionTextareaVisible
         ? this.handleDescriptionUpdate
         : this.handleNameUpdate;
 
       action();
     }
```
Comment (marekrozmus) [inline client/src/modules/list/components/ListHeader.jsx]: return as in previous comment

File: client/src/modules/list/components/ListHeader.jsx [hunk 4]
```diff
@@ -91,18 +95,20 @@ class ListHeader extends PureComponent {
...
 
-    if (
-      isNameInputVisible &&
-      _trim(nameInputValue).length >= 1 &&
-      isClickedOutside
-    ) {
-      this.handleNameUpdate();
+    if (isNameInputVisible && isClickedOutside) {
+      if (_trim(nameInputValue).length >= 1) {
```
Comment (marekrozmus) [inline client/src/modules/list/components/ListHeader.jsx]: > 0

File: client/src/modules/cohort/components/CohortHeader.jsx [hunk 5]
```diff
@@ -48,43 +48,49 @@ class CohortHeader extends PureComponent {
...
+    if (code === KeyCodes.ENTER) {
+      this.handleNameUpdate();
     }
   };
 
-  handleNameInputVisibility = () =>
-    this.setState(({ isNameInputVisible }) => ({
-      isNameInputVisible: !isNameInputVisible
-    }));
+  handleShowNameInput = () =>
```
Comment (adamklepacz) [inline client/src/modules/cohort/components/CohortHeader.jsx]: just 
showNameInput

File: client/src/modules/cohort/components/CohortHeader.jsx [hunk 6]
```diff
@@ -48,43 +48,49 @@ class CohortHeader extends PureComponent {
...
+    this.setState({
+      isNameInputVisible: true
+    });
 
-  handleDescriptionTextareaVisibility = () =>
-    this.setState(({ isDescriptionTextareaVisible }) => ({
-      isDescriptionTextareaVisible: !isDescriptionTextareaVisible
-    }));
+  handleShowDescriptionTextarea = () =>
```
Comment (adamklepacz) [inline client/src/modules/cohort/components/CohortHeader.jsx]: just 
showDescriptionTextarea

File: client/src/modules/list/components/ListHeader.jsx [hunk 7]
```diff
@@ -44,15 +44,15 @@ class ListHeader extends PureComponent {
     }
   }
 
-  handleNameInputVisibility = () =>
-    this.setState(({ isNameInputVisible }) => ({
-      isNameInputVisible: !isNameInputVisible
-    }));
+  handleShowNameInput = () =>
```
Comment (adamklepacz) [inline client/src/modules/list/components/ListHeader.jsx]: showNameInput

File: client/src/modules/list/components/ListHeader.jsx [hunk 8]
```diff
@@ -44,15 +44,15 @@ class ListHeader extends PureComponent {
...
+    this.setState({
+      isNameInputVisible: true
+    });
 
-  handleDescriptionTextareaVisibility = () =>
-    this.setState(({ isDescriptionTextareaVisible }) => ({
-      isDescriptionTextareaVisible: !isDescriptionTextareaVisible
-    }));
+  handleShowDescriptionTextarea = () =>
```
Comment (adamklepacz) [inline client/src/modules/list/components/ListHeader.jsx]: showDescriptionTextarea

**Pull request merged**